app/controllers/sessions_controller.rb

```diff
@@ -1,9 +1,9 @@
 class SessionsController < Devise::SessionsController
   def destroy
     # Let's say goodbye to all the cookies when someone signs out.
-    cookies.each do |cookie|
-      cookies.delete cookie[0]
-    end
+    domain = Rails.env.production? ? ApplicationConfig["APP_DOMAIN"] : nil
```

Comment:
Interesting, I think we would need this on development too. Not sure if `localhost:3000` behaves differently but I can reproduce the problem in development with a custom domain (ngrok or xip.io for mobile development)

Reply:
We don't see the domain in development: 

https://github.com/forem/forem/blob/6c2357af48e801b4d403accbbb685958d24a8f79/config/initializers/session_store.rb#L12-L13

Reply:
Oh cool! Defenitely didn't see that